client/src/main/java/com/vesoft/nebula/client/meta/MetaInfo.java

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.client.meta;
+
+import com.google.common.collect.Maps;
+import com.google.common.net.HostAndPort;
+import com.vesoft.nebula.meta.EdgeItem;
+import com.vesoft.nebula.meta.TagItem;
+import java.util.List;
+import java.util.Map;
+
+public class MetaInfo {
+
+    private final Map<String, Integer> spaceNameMap = Maps.newHashMap();
```

Comment:
you can define a class to save all the space info,  and not need so many key to save the space.

such as:
```
class SpaceInfo {
      spaceItem;
      tagItems;
      edgeItems;
      partInfo;
      ……
}

map<string, spaceInfo>
```